test/query.test.js

```diff
@@ -1906,17 +1936,19 @@ describe('query', () => {
   test('and', () => {
     var p1 = assertQuery(query.And(true, true, false), false)
     var p2 = assertQuery(query.And(true, true, true), true)
-    var p3 = assertQuery(query.And(true), true)
-    var p4 = assertQuery(query.And(false), false)
-    return Promise.all([p1, p2, p3, p4])
+    // var p3 = assertQuery(query.And(true), true)
```

Comment:
These are commented out for now but will be re-added in the future: https://github.com/fauna/faunadb-js/pull/295